media/custom/templates/libraries.html

```diff
@@ -0,0 +1,308 @@
+{% extends "base_for_backbone.html" %}
```

Comment:
This whole file is duplicated from `seahub/templates/libraries.html` to only change one line ("Welcome to Seafile!" => "Welcome to Open365"). Cannot we extract the name of the product into a variable and use it like it is using `{{ SITE_ROOT }}` in line 21 (and remove this file)?